### ISSUE TITLE
php form won't load anything

### ISSUE THREAD

**Issue by wolftone1916** [formtools/core#295]
So I have a couple of forms I'm using but all are coded in HTML and I'm trying to get them coded in php to better use the captcha feature as I've been getting spam in the forums. When I go through the tutorial, I get a 500 error (talking to hosting company about that). When I move the code away from it, I just get a blank page. The formtools install is sitting in the directory contactus so when you see the coding below, that's the directory start to get to the API. I'm using formtools version: 3.0.4  that is installed through softaculous.

Below is the page I'm trying to do it on
[http://www.southernstarbrewing.com/contactusform.php](http://www.southernstarbrewing.com/contactusform.php)

**Comment by wolftone1916** [comment]
Sorry, more crucial info that could help here:
PhP version: 7.1.20
And attached is the actual API file that it is trying to load.
[API.class.txt](https://github.com/formtools/core/files/2280540/API.class.txt)

**Comment by wolftone1916** [comment]
Okay so, figured out what I was doing wrong so in case anyone else has an issue, they could use this to see what that issue was. I had to enable php error logging to show on the site what the errors were. I did that and found that there were 2 errors causing problems. I realized that the directory is somewhat wonky in how it was calling to things like the api. Fixed the first error (basically unicode bit) and then fixed the correct directory (by looking at a form I built for testing on the form builder). Now it's working.
